pipeline/BuildBaseImage/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM nodesource/wheezy:4.4.6
+
+RUN apt-get update && apt-get install -y \
```

Comment:
thank you @PengTian0 .
as a PR Gate role model, if making the BuildBaseImage job/pipeline as a option , and the code is very easy to switch between the two modes( build from scratch v.s. build from baseImg ), it will be great !